src/test/config/bitbucket/values.yaml

```diff
@@ -2,6 +2,9 @@
 
 replicaCount: ${replica.count}
 
+image:
+  tag: 8.0.0
```

Comment:
This was just for mesh testing purposes. It is better to not pin the version for general testing.